### PR TITLE
start updating tutorial pages and Solidity to Cadence page to Stable Cadence

### PR DESCRIPTION
**Pull request by turbolent**
Depends on #266 

Mostly focusing on updating API usage.

There are many more things that need to be adjusted in these pages.

### CODE REVIEW

File: docs/cadence/tutorial/05-non-fungible-tokens-2.mdx [hunk 1]
```diff
@@ -604,7 +604,8 @@ transaction {
     prepare(acct: AuthAccount) {
```
Comment (SupunS) [inline docs/cadence/tutorial/05-non-fungible-tokens-2.mdx:604]:
```suggestion
    prepare(acct: auth(BorrowValue) &Account) {
```

File: docs/cadence/tutorial/06-fungible-tokens.mdx [hunk 2]
```diff
@@ -478,7 +478,7 @@ transaction {
   prepare(acct: AuthAccount) {
```
Comment (SupunS) [inline docs/cadence/tutorial/06-fungible-tokens.mdx:478]:
```suggestion
  prepare(acct: auth(BorrowValue) &Account) {
```

File: docs/cadence/tutorial/06-fungible-tokens.mdx [hunk 3]
```diff
@@ -478,7 +478,7 @@ transaction {
...
     // withdraw tokens from your vault by borrowing a reference to it
     // and calling the withdraw function with that reference
-    let vaultRef = acct.borrow<&BasicToken.Vault>(from: /storage/CadenceFungibleTokenTutorialVault)
+    let vaultRef = acct.storage.borrow<&BasicToken.Vault>(from: /storage/CadenceFungibleTokenTutorialVault)
```
Comment (SupunS) [inline docs/cadence/tutorial/06-fungible-tokens.mdx:481]: Think this should borrow a `Withdrawable` reference
```suggestion
    let vaultRef = acct.storage.borrow<auth(FungibleToken.Withdrawable) &BasicToken.Vault>(from: /storage/CadenceFungibleTokenTutorialVault)
```

**Review by SupunS — APPROVED** [review]

**Comment by turbolent** [comment]
Note that this PR only _starts_ to update the tutorial pages and Solidity to Cadence page, there is much more to do to also adjust the contents accordingly

**Pull request merged**